### DISCUSSION TITLE
Are comments allowed in SQL queries?

### DISCUSSION THREAD

**Discussion by dnaeye** [mkleehammer/pyodbc#1104] [Q&A]
Is there a way to run a query with comments indicated by "--" syntax?

**Comment by David-Engel** [comment]
I believe so. Did you try it and encounter an error?

**Comment by v-chojas** [comment]
Depends on your DBMS and driver - what are you using? pyODBC just passes through queries.

**Comment by mkleehammer** [comment]
In general, yes.  I've used them with SQL Server, MySQL, and PostgreSQL.
